cfgov/unprocessed/apps/rural-or-underserved-tool/js/address-utils.js

```diff
@@ -1,5 +1,4 @@
 import DT from './dom-tools';
-const fullCountyList = require( '../data/counties.json' );
```

Comment:
Can https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/rural-or-underserved-tool/data/counties.json be deleted?